### ISSUE TITLE
Question about the final presentation

### ISSUE THREAD

**Issue by jhwest1** [snu-sf-class/swpp202401#174]
Hi, is the final presentation also lightweight as the plan presentation from the beginning of the project?

In other words, can we focus on making jokes and try to be funny (but we will still share our experiences during the project), instead of being serious, explaining the technical details of our optimization, etc?

**Comment by strikef** [comment]
Yes, the final presentation doesn't have to be serious, just like how we did at the first team presentation.
In fact, each team will only have around 5 minutes to present, so I doubt there will be enough time to explain all the technical details about the optimization passes (although it would be exciting)
